app/models/health/comprehensive_health_assessment.rb

```diff
@@ -893,10 +899,12 @@ def self.answers_for section: nil, question: nil, number: 0
 
     scope :recent, -> { order(updated_at: :desc).limit(1) }
     scope :reviewed, -> { where.not(reviewed_by_id: nil) }
+    scope :reviewed_within, ->(range) { where(reviewed_at: range) }
     scope :incomplete, -> { where(completed_at: nil, reviewed_by_id: nil) }
     # hide previous declaration of :complete, we'll use this one
     replace_scope :complete, -> { where.not(completed_at: nil) }
     scope :completed, -> { complete }
+    scope :completed_within, ->(range) { where(completed_at: range) }
```

Comment:
We've seen some issues when comparing timestamps to date ranges, should we convert the range to a timestamp range first?